explorer/nodewatch/nodewatch/templates/symbol_nodes.html

```diff
@@ -66,10 +67,42 @@ <h2 class="text-center py-4">{{ title }}</h2>
         {% endif %}
 
         {% if show_voting %}
-        <td>{{ descriptor.voting_end_epoch }}</td>
+        <td class="voting-end-epoch">{{ descriptor.voting_end_epoch }}</td>
+        <td class="voting-expiry">{{ descriptor.voting_expiry }}</td>
         {% endif %}
       </tr>
       {% endfor %}
     </tbody>
   </table>
 {% endblock %}
+
+{% block scripts %}
+  <script type="text/javascript">
+    function update_voting_key_expiry() {
+      fetch('{{ url_for("api_symbol_epoch") }}', {headers: {'Content-Type': 'application/json'} })
+        .then(response => response.json())
+        .then(response => {
+          const currentEpoch = response['epoch'];
+
+          // Update each row's expiry days
+          document.querySelectorAll('tbody tr').forEach(row => {
+            const votingEndEpoch = row.querySelector('.voting-end-epoch').textContent;
+            const expiryCell = row.querySelector('.voting-expiry');
+
+            if (votingEndEpoch && expiryCell) {
+              const blocksPerDay = 2880;
+              const blocksPerEpoch = 1440;
```

Comment:
These values can/will be different between testnet and mainnet.

Reply:
After some thought, `miscellaneous` might not be the best option because we only need the network config once when starting the instance. Instead, I added it into the app config.

here's the commit [e2608f6](https://github.com/symbol/product/pull/1433/commits/e2608f653cc7982a3fe0d934b111d657e3a8b14e)